### PR TITLE
Increase camera figures in how does camera work article.

### PR DESCRIPTION
**Pull request by anaritaflp**
This PR increases the images in the `how-does-the-pickit-3d-camera-work` article.

It does not center the images - this is done in https://github.com/Intermodalics/pickit-docs/pull/26.

Before:
image

After:
image

### CODE REVIEW

**Comment by anaritaflp** [comment]
For some reason, if the names of old and new figure are the same, it does not update the figure. Possibly some optimization being done, but I don't know how to "refresh". Restarting the docker instance and refreshing the page did not work.

**Comment by anaritaflp** [comment]
@PhilColpaert @adolfo-rt Indeed, I forgot to push the changes on the article. Thanks for noticing. It's pushed now.

File: docs/faq/general/how-does-the-pickit-3d-camera-works.rst [hunk 1]
```diff
@@ -3,68 +3,74 @@
...
-For the Pick-it M and L cameras, :ref:`image-fusion` leads to a more stable point cloud. 
-For the Pick-it M-HD camera, a different camera preset can be used. 
-If still not enough 3D is obtained the parts are too reflective or transparent. 
\ No newline at end of file
+The above scenarios exemplify edge cases.
+Often parts are only partially reflective or semi-transparent.
+When in doubt about a part, it is recommended to test it by placing it under the camera, trigger a detection and inspect the point cloud in the **Points** tab of the :ref:`Viewer`.
+If not enough 3D data is shown, know that this can be further optimized.
+For the Pick-it M and L cameras, :ref:`image-fusion` leads to a more stable point cloud.
+For the Pick-it M-HD camera, a different camera preset can be used.
```
Comment (PhilColpaert) [inline docs/faq/general/how-does-the-pickit-3d-camera-works.rst]: Add reference to article:

:ref:`how-to-mhd-preset`
